devel/create_distro_tarball.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/sh
+#
+# Copyright 2011 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: morlovich@google.com (Maksim Orlovich)
+#
+# The usual mechanism used to develop mod_pagespeed and build binaries is based
+# on merging all dependencies into a single source tree.  This script enables a
+# standard  untar/configure/make flow that does not bundle widely available
+# external libraries. It generates the tarball including the configure (or
+# rather generate.sh) script.
+#
+# If --minimal is passed, it will cut out even more things. This was meant
+# for packaging properly Debian, which has a particularly extensive package
+# repository. At the moment this configuration requires further patching of
+# the .gyp[i] files and doesn't work out of the box. The pruning was also done
+# as of branch 33, so further tweaks might be required for this mode in
+# 34 or newer.
+
+set -e  # exit script if any command returns an error
+set -u  # exit the script if any variable is uninitialized
+
+function usage {
+  echo -n "create_distro_tarball_debian.sh [ --minimal ] [ --checkout_dir dir ] tarball"
+  echo " [ --git_tag tag | source_tree ]"
```

Comment:
I think maybe I'd say '--checkout commit_ish' which is the git way of saying "something that can be checked out.